### ISSUE TITLE
FirebaseInstallations NullPointException

### ISSUE THREAD

**Issue by zubinxiong** [firebase/firebase-android-sdk#1795]
###  Describe your environment

  * Android Studio version: 4.0
  * Firebase Component: firebase-analytics:17.4.3 ， firebase-messaging:20.2.1

#### Steps to reproduce:

I can't reproduce about this bug, it's collected by bug report sdk, only some devices have this bug

#### Relevant Code:

```Java
 firebase-installations-executor-٥(620)
 java.lang.NullPointerException
 Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
 com.google.firebase.installations.i.d(FirebaseInstallations.java:2)
 com.google.firebase.installations.i.c(FirebaseInstallations.java:6)
 com.google.firebase.installations.i.b(FirebaseInstallations.java:3)
 com.google.firebase.installations.g.run(Unknown Source:2)
 java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
 java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
 java.lang.Thread.run(Thread.java:764)

```

**Comment by ankitaj224** [comment]
@zubinxiong Thanks for reporting this issue. This is a duplicate of #1787. Can you please follow up there?

It has been fixed addressed in #1714 but the fix hasn't been released yet. We will keep the other issue posted when we release the latest versions.

**Comment by zubinxiong** [comment]
@ankitaj224 thanks，I will follow up the problem in #1787
